packages/forms-web-app/src/locales/cy/common.json

```diff
@@ -10,6 +10,9 @@
 	"next": "Nesaf",
 	"showAllSteps": "Dangos pob cam",
 	"hideAllSteps": "Cuddio pob cam",
+	"resultsPerPage": "Canlyniadau fesul tudalen",
+	"clearFilters": "Clear filters",
```

Comment:
Should clear filters and select all filters be in Welsh?

Reply:
We are still waiting on a number of translations that are missing, these are noted in the comments section on the ticket. I will hold off merging until these can be updated.